web-app-dsc-vm/azuredeploy.json

```diff
@@ -0,0 +1,423 @@
+            "defaultValue": "Subnet-2",
+            "metadata": { 
+                "Description": "This is the name of the second subnet" 
+            }
+        },
+        "subnet1Prefix": {
+            "type": "string",
+            "defaultValue": "10.0.0.0/24",
+            "metadata": { 
+                "Description": "This is the address prefix for the first subnet" 
+            }
+        },
+        "subnet2Prefix": {
+            "type": "string",
+            "defaultValue": "10.0.1.0/24",
+            "metadata": { 
+                "Description": "This is the address prefix for the second subnet" 
+            }
+        },
+        "dnsName": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the DNS name of the Virtual Machine" 
+            }
+        },
+        "subscriptionId": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the subscription ID under which the resource group will be created" 
+            }
+        },
+        "nicName": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the name of the Network Interface of the Virtual Machine" 
+            }
+        },
+        "vmExtensionName": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the name of the DSC extension that will be applied to the Virtual Machine" 
+            }
+        },
+        "modulesUrl": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the URL for downloading the PowerShell DSC module. In this case it should be the link to a zip file hosted under an Azure storage container" 
+            }
+        },
+        "sasToken": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the Shared Access Signature token used for accessing the Azure storage URL linked to the PowerShell DSC module" 
+            }
+        },
+        "webdeploypkg": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the link to the Web Deploy package for the website that's going to be deployed to the Virtual Machine" 
+            }
+        },
+        "configurationFunction": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the link to the Web Deploy package to be deployed to the Virtual Machine. It is a github URL in this example." 
+            }
+        },
+        "serverName": {
+            "type": "string",
+             "metadata": { 
+                "Description": "This is database server name" 
+            }
+        },
+        "serverLocation": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is database server location" 
+            }
+        },
+        "administratorLogin": {
```

Comment:
Probably should be something like dbAdminName to be clearly different from the the vm admin. Or could be we combine the two parameters into one and reuse them?